source/NVDAObjects/window/excel.py

```diff
@@ -505,27 +508,43 @@ def navigationHelper(self,direction):
 		cellPosition.Activate()
 		eventHandler.executeEvent('gainFocus',obj)
 
+	@script(
+		gesture="kb:leftArrow")

```

Comment:
There is no need to place single parameter on its own line unless linte complains about line length. In other words `@script(gesture="kb:leftArrow")` is fine and reduces the diff considerably.